### PR TITLE
fix: get all explores in one query for dashboard available filters

### PR DESCRIPTION
**Pull request by ZeRego**
Closes: 

### Description:

Query all explores in 1 query (removes n+1 query)

There is a separate [tech debt ticket](https://github.com/lightdash/lightdash/issues/10384) to replace all the old model methods with the new one.

test-frontend

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/backend/src/services/ProjectService/ProjectService.ts [hunk 1]
```diff
@@ -2457,62 +2457,97 @@ export class ProjectService extends BaseService {
             description: 'Gets a single explore from the cache',
```
Comment (joaoviana) [inline packages/backend/src/services/ProjectService/ProjectService.ts:2457]: will review this soon! When you get latest Sentry, could you change this `Sentry` work here? Should be similar to: 
https://github.com/lightdash/lightdash/blob/main/packages/backend/src/services/ProjectService/ProjectService.ts#L2452

```ts
 return Sentry.startSpan(
            {
                op: 'ProjectService.getExplore',
                name: 'ProjectService.getExplore',
            },
            async () =>
                wrapOtelSpan('ProjectService.getExplore', {}, async () => {
....

```

no need to `.finish()` anymore as well

Comment (ZeRego) [reply packages/backend/src/services/ProjectService/ProjectService.ts]: done ✅

**Comment by joaoviana** [comment]
Could you trigger e2e tests for frontend @ZeRego ? 
I'm almost done with reviewing/testing this.

File: packages/backend/src/services/ProjectService/ProjectService.ts [hunk 2]
```diff
@@ -2669,39 +2718,30 @@ export class ProjectService extends BaseService {
...
+                    await Promise.all([
+                        this.spaceModel.getSpacesForAccessCheck(
+                            uniqueSpaceUuids,
+                        ),
+                        this.findExplores({
+                            user,
+                            projectUuid: savedCharts[0].projectUuid, // TODO: route should be updated to be project/dashboard specific. For now we pick it from first chart as they all should be from the same project
+                            exploreNames: savedCharts.map(
+                                (chart) => chart.tableName,
+                            ),
```
Comment (joaoviana) [inline packages/backend/src/services/ProjectService/ProjectService.ts:2736]: can we pass `organizationUuid` from the `user.organizationUuid` here?

**Review by joaoviana — APPROVED** [review]

File: packages/backend/src/services/ProjectService/ProjectService.ts (hunk 2 above)
Comment (ZeRego) [reply packages/backend/src/services/ProjectService/ProjectService.ts:2736]: there is no `organizationUuid` in the code

Comment (ZeRego) [reply packages/backend/src/services/ProjectService/ProjectService.ts:2736]: ah, got it 👍 thanks, adding it now

**Pull request merged**